R/test-DataClass.R

```diff
@@ -102,7 +102,8 @@ test_download <- function(DataClass_obj, download, snapshot_path) {
         walk(DataClass_obj$data$raw, function(data) {
           testthat::expect_s3_class(data, "data.frame")
           testthat::expect_true(nrow(data) > 0)
-          testthat::expect_true(ncol(data) >= 2)
+          testthat::expect_true(ncol(data) >= 2
+                                || typeof(data[[1]])=="list")
```

Comment:
This adjustment lets us deal with cases where we've downloaded a JSON file and ended up with a tibble full of lists. Without starting to do a lot of cleaning in the `download` method I think this is a good compromise on setting up a test which will catch true if the number of columns is less than 2.